### PR TITLE
WIP: Proof of concept calendar embed

### PR DESCRIPTION
**Pull request by ewels**
Calendar styling is all done within https://styledcalendar.com
Calendar events are handled within Google Calendar

Proof of concept only:
* Haven't added all events yet
* Nothing for Boston
* Boston & Barcelona pages show the same calendar

### CODE REVIEW

File: src/pages/barcelona/agenda/calendar.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              <Image
+                image={getImage(data.heroImgBarcelona)}
+                alt="Join us in Barcelona or virtually"
+                className="rounded-sm shadow-xl"
+                imgClassName="rounded-sm"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
```
Comment (MythicalFish) [inline src/pages/barcelona/agenda/calendar.jsx:58]: Do we want these intro sections here too? I'd be tempted to just have the calendar itself, and a back button at the top, which goes back to `/barcelona/agenda` 🤔

File: src/pages/barcelona/agenda/calendar.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <div className="container-lg text-white pt-36 pb-16" id="events">
+        <h2 className="typo-h5 uppercase text-center mb-4">Agenda</h2>
+        <div className="calendar-container">
+          <iframe src="https://embed.styledcalendar.com/#Qss4ZvPVKjMoFYIc82hS" title="Styled Calendar" className="styled-calendar-container w-full border-0 h-[600px]" data-cy="calendar-embed-iframe"></iframe>
```
Comment (MythicalFish) [inline src/pages/barcelona/agenda/calendar.jsx:62]:
```suggestion
          <iframe src="https://embed.styledcalendar.com/#Qss4ZvPVKjMoFYIc82hS" title="Barcelona SUMMIT 2023 Calendar" className="styled-calendar-container w-full border-0 h-[600px]" data-cy="calendar-embed-iframe"></iframe>
```

File: src/pages/boston/agenda/calendar.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                imgClassName="rounded-sm"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="container-lg text-white pt-36 pb-16" id="events">
+        <h2 className="typo-h5 uppercase text-center mb-4">Agenda</h2>
+        <div className="calendar-container">
+          <iframe src="https://embed.styledcalendar.com/#A5RyQEwveb0zYn69ncqd" title="Styled Calendar" className="styled-calendar-container w-full border-0 h-[600px]" data-cy="calendar-embed-iframe"></iframe>
```
Comment (MythicalFish) [inline src/pages/boston/agenda/calendar.jsx:62]:
```suggestion
          <iframe src="https://embed.styledcalendar.com/#A5RyQEwveb0zYn69ncqd" title="Boston SUMMIT 2023 Calendar" className="styled-calendar-container w-full border-0 h-[600px]" data-cy="calendar-embed-iframe"></iframe>
```

**Review by MythicalFish — APPROVED** [review]
LGTM, just some minor suggestions. :) You could create a reusable component for the calendar which lives under modules (i.e. `modules/CalendarPage`), but probably not worth the effort unless we're using this same codebase next year

File: src/pages/barcelona/agenda/calendar.jsx (hunk 1 above)
Comment (ewels) [reply src/pages/barcelona/agenda/calendar.jsx:58]: Sure, like the call for abstract instructions? Works for me 👍🏻

Comment (ewels) [reply src/pages/barcelona/agenda/calendar.jsx]: Done ✅

**Comment by ewels** [comment]
Yeah, I'm not convinced that we'll use this calendar setup again again next year. And the site might be totally different. So I'll leave that for another day :)

Thanks for the review / suggestions!

**Pull request merged**